tests/db/Dockerfile.mssql

```diff
@@ -19,9 +19,3 @@ RUN apt-get update && ACCEPT_EULA=Y apt-get install -y mssql-tools unixodbc-dev
 RUN pip install dist/*.whl
 RUN pip install pyodbc
 RUN pip list
-
-COPY log.py .
-COPY init-mssql-db.sh .
-COPY init-mssql-db.sql .
-
-RUN chmod +x init-mssql-db.sh
```

Comment:
We no longer need these line since we mount `tests/db` when running the container.